styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+}
+
+body {
+    background: url(./images/OpLogo.png);
+    background-size: 15vh;
+}
+
+.sContainer {
+    width: 100vw;
+    height: 100vh;
+
+}
+.crew_title {
+    font-size: 5vw;
+    background-color: yellow;
+    color:blue;
+    text-align: center;
+    margin-bottom: 5px;
+    padding-bottom: 5px;
+}
+
+.crew_button {
+    background-color: red;
+    color: blue;
+    font-size: 5vw;
+    text-decoration: underline;
+    padding: 1vh 1vw;
+    border-radius: 5vw;
+    font-size: 2vh;
+    font-weight: 860;
+
+}
+
+.crew_description {
+    font-size: 5vw;
+    display: none;
+/* hid the description... */
```

Comment:
I like this idea of hiding the description and toggling it on. Makes for a cleaner website.